core/src/main/java/com/netease/arctic/scan/TableEntriesScan.java

```diff
@@ -61,9 +62,11 @@ public class TableEntriesScan {
   private final Expression dataFilter;
   private final boolean aliveEntry;
   private final boolean allFileContent;
+  private final boolean includeColumnStats;
   private final Set<FileContent> validFileContent;
 
   private Table entriesTable;
+  private PartitionSpec spec;
```

Comment:
By using `final` to describe the spec and initializing it in this constructor, we can avoid calling the function. 
What do you think is it better to initialize immutable variables immediately or lazily load?

Reply:
Honestly, they do not have much difference in this situation, but I have a preference to keep the constructor simple.